cypress/suites/dashboard.suite.ts

```diff
@@ -122,27 +177,383 @@ export default {
+        it("should execute a buy properly", () => {
+          cy.get("div").find(">label").parent().type(validDexWidgetSearchByAssetId);
+          cy.contains(validSearchByAssetIdResult).click();
+          cy.get('input[placeholder*="Price"]').type("1");
+          cy.get('input[placeholder*="Quantity"]').eq(0).type("1");
+
+          cy.get("button").contains("Buy").click();
+          cy.get('input[placeholder*="Enter Seed Phrase"]').type(testnetSeedPhrase);
+          cy.get("button").contains("Confirm & Send").click();
+          cy.get("#notistack-snackbar").should("contain.text", messageText.orders.success); // snackbar
+        });
+
+        // order message is not correct, currently it's messageText.transaction.success which must be a defect in the useAPIRouter()
+        it("should execute a sell properly", () => {
+          cy.get("div").find(">label").parent().type(validDexWidgetSearchByAssetId);
+          cy.contains(validSearchByAssetIdResult).click();
+          cy.get('input[placeholder*="Price"]').type("1");
+          cy.get('input[placeholder*="Quantity"]').eq(0).type("1");
+
+          cy.get("button").contains("Sell").click();
+          cy.get('input[placeholder*="Enter Seed Phrase"]').type(testnetSeedPhrase);
+          cy.get("button").contains("Confirm & Send").click();
+          cy.get("#notistack-snackbar").should("contain.text", messageText.orders.success); // snackbar
+        });
+
+        it("should fire cancellation snackbar message", () => {
+          cy.get("div").find(">label").parent().type(validDexWidgetSearchByAssetId);
+          cy.contains(validSearchByAssetIdResult).click();
+          cy.get('input[placeholder*="Price"]').type("1");
+          cy.get('input[placeholder*="Quantity"]').eq(0).type("1");
+
+          cy.get("button").contains("Sell").click();
+          cy.get("button").contains("Done").click();
+          cy.get("#notistack-snackbar").should("contain.text", messageText.transaction.cancel); // snackbar
+        });
+      });
+
+      describe("user info", () => {
+        beforeEach(() => {
+          cy.visit("/");
+          existingUserLogin();
+        });
+
+        // test doesn't work in Firfox
+        it("should provide a notification on address copy", () => {
+          cy.get(".MuiDrawer-root > .MuiPaper-root > :nth-child(1) > .MuiChip-sizeMedium").click();
+          cy.get("#notistack-snackbar").should("contain.text", messageText.copy.success); // snackbar
+        });
+
+        // test doesn't work in Firfox
+        it("should provide a notification on balance copy", () => {
+          cy.get(".css-18w7urt-MuiButtonBase-root-MuiChip-root > .MuiChip-label").click();
+          cy.get("#notistack-snackbar").should("contain.text", messageText.copy.success); // snackbar
+        });
+
+        it("should open user info dialog", () => {
+          cy.get(".css-v2w70v-MuiButtonBase-root-MuiChip-root > .MuiChip-label").click();
```

Comment:
noted, i'm going to leave as-is for now but it's good to have it in mind because if it starts failing it should be more obvious to me why it's happening

I want to get a lot better with the selectors overall but i'm just not there yet